self-api/src/main/java/com/selfxdsd/api/exceptions/RepoAlreadyActiveException.java

```diff
@@ -0,0 +1,21 @@
+package com.selfxdsd.api.exceptions;
+
+import com.selfxdsd.api.Repo;
+
+/**
+ * Exception thrown if we try to active an already active Repo.
+ * @author criske
+ * @version $Id$
+ * @since 0.0.20
+ */
+public final class RepoAlreadyActiveException extends RuntimeException {
+
+    /**
+     * Ctor.
+     * @param repo Repo in question.
+     */
+    public RepoAlreadyActiveException(final Repo repo){
+        super("Repo \"" + repo.fullName() + "\" is already active.");
```

Comment:
I've tried to keep the same pattern as `InvoiceAlreadyPaidException`.
But in this case `repo.fullName()` will require another json request. So unless we find a way to cache these json responses (etags or something) -- or at least part of them like `fullName`, maybe we should pass the `fullName` string instead. (and possibly the `Provider` name, just to make the message clearer.)

Reply:
@criske Method [BaseRepo.json()](https://github.com/self-xdsd/self-core/blob/master/self-core-impl/src/main/java/com/selfxdsd/core/BaseRepo.java#L91) caches the json so there will be only one initial real call made, right? 

Or am I missing something? :D

Reply:
You're right, dunno why I had the impression it didn't lol. You can merge it